firewood/src/merkle/node.rs

```diff
@@ -133,7 +133,7 @@ impl BranchNode {
                 Some(c) => {
                     let mut c_ref = store.get_item(*c).unwrap();
 
-                    if c_ref.is_encoded_big::<S>(store) {
+                    if c_ref.is_encoded_longer_than_hash_len::<S>(store) {
```

Comment:
You don't need the `len` on the end because `longer` implies length...

Reply:
True, but "Longer than hash" is less readable IMO.